cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -33,6 +33,9 @@ const (
 	DefaultCACertPath        = "/etc/kubernetes/pki/ca.crt"
 	DefaultCertificatesDir   = "/etc/kubernetes/pki"
 	DefaultEtcdDataDir       = "/var/lib/etcd"
+	DefaultEtcdClusterSize   = 3
+	DefaultEtcdOperatorCount = 1
+	DefaultEtcdVersion       = "v3.1.4"
```

Comment:
I imagine a user wanting to override the etcd version will be quite common, whereas the image URL will remain fairly static. Is that what you meant?